AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant-base/add-participant-base.component.ts

```diff
@@ -198,9 +198,9 @@ export abstract class AddParticipantBaseDirective extends BookingBaseComponent i
         this.displayErrorNoParticipants = false;
         this.displayAdd();
         this.enableFields();
-        this.participantDetails = { ...participantDetails };
+        this.participantDetails = participantDetails.clone();
```

Comment:
Why use a clone method? Could the object spread operator such as:
```suggestion
        this.participantDetails = {...participantDetails};
```

Reply:
I had this originally, but unfortunately the spread operator doesn't clone the getters